server/cluster/cluster.go

```diff
@@ -295,11 +297,74 @@ func (c *RaftCluster) Start(s Server) error {
 	go c.runMinResolvedTSJob()
 	go c.runSyncConfig()
 	go c.runUpdateStoreStats()
+	go c.startGCTuner()
 
 	c.running.Store(true)
 	return nil
 }
 
+// startGCTuner
+func (c *RaftCluster) startGCTuner() {
+	defer logutil.LogPanic()
+	defer c.wg.Done()
+
+	tick := time.NewTicker(10 * time.Second)
+	defer tick.Stop()
+	totalMem, err := memory.MemTotal()
+	if err != nil {
+		log.Fatal("fail to get MemTotal:%s", zap.Error(err))
```

Comment:
Does it need to be fatal?

Reply:
I think that's OK, see one of the usage in TiDB: https://github.com/pingcap/tidb/blob/3f0964ff9dd6cfc11b956f0835af85cd5ed4206e/tidb-server/main.go#L749